engine/runtime/src/main/scala/org/enso/interpreter/runtime/SerializationManager.scala

```diff
@@ -401,6 +404,7 @@ final class SerializationManager(private val context: TruffleCompilerContext) {
     compiler: Compiler,
     module: Module
   ): Option[Boolean] = {
+    compiler.getClass()
```

Comment:
?

Reply:
Without that code to Scala compiler complains:
```
SerializationManager.scala:400:5: parameter compiler in method deserialize is never used
[error]     compiler: Compiler,
```